src/CoreMetadata/page.jsx

```diff
@@ -20,7 +20,7 @@ class CoreMetadataPage extends Component {
   render() {
     return (
       <div>
-        <BackLink url='/files' label='Back to data overview' />
+        <BackLink url="/files" label="Back to data overview" />
```

Comment:
We're supposed to use single-quotes in javascript, and double quotes in jsx?  Ugh.

Reply:
if there are conflicting styles I can try to standardize it

Reply:
Is there one we want? I can set that in ESLint so they are the same

Reply:
Standardized it to single - so there are a lot of changed files now